### PR TITLE
Ca feature/yml env files support

### PR DESCRIPTION
**Pull request by cagodoy**
Added support for using yml configuration file to pigeon service.
it is also possible to define the path of the configuration file using the `--path`

### CODE REVIEW

File: cmd/scheduler/main.go [hunk 1]
```diff
@@ -3,45 +3,84 @@ package main
...
 
-func main() {
-	port := flag.Int("port", 5050, "port of the service")
-	host := flag.String("host", "", "host of the service")
-	dbfile := flag.String("db", "massages.db", "file to store messages")
+// Config define all env values used for service execution
+type Config struct {
+	Port             string `yaml:"port"`
+	Host             string `yaml:"host"`
+	DbFile           string `yaml:"db_file"`
```
Comment (jaguirresza) [inline cmd/scheduler/main.go:24]: s/DbFile/DBFile/

File: cmd/scheduler/main.go [hunk 2]
```diff
@@ -3,45 +3,84 @@ package main
...
+	RedisURL         string `yaml:"redis_url"`
+	RedisIdleTimeout string `yaml:"redis_idle_timeout"`
```
Comment (jaguirresza) [inline cmd/scheduler/main.go:26]: Why is this a string?

File: cmd/scheduler/main.go [hunk 3]
```diff
@@ -3,45 +3,84 @@ package main
...
+	yaml "gopkg.in/yaml.v2"
 )
 
-func main() {
-	port := flag.Int("port", 5050, "port of the service")
-	host := flag.String("host", "", "host of the service")
-	dbfile := flag.String("db", "massages.db", "file to store messages")
+// Config define all env values used for service execution
+type Config struct {
+	Port             string `yaml:"port"`
```
Comment (jaguirresza) [inline cmd/scheduler/main.go:22]: Why is this a string?

**Review by jaguirresza — CHANGES_REQUESTED** [review]

File: cmd/scheduler/main.go (hunk 1 above)
Comment (cagodoy) [reply cmd/scheduler/main.go:24]: 🐒

Comment (cagodoy) [reply cmd/scheduler/main.go:24]: @jorgeaguirre  tontito 🐒

Comment (Weyker) [reply cmd/scheduler/main.go:24]: aaa camilo, ahora cache lo q qeria decir... xD

q cambies DbFile por DBFile

mucho vim el ql

**Pull request closed without merging**